Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -213,14 +206,15 @@ public User GetUserByName(string name)
                 throw new ArgumentNullException(nameof(name));
             }
 
-            return Users.FirstOrDefault(u => string.Equals(u.Name, name, StringComparison.OrdinalIgnoreCase));
+            return _users.Values.FirstOrDefault(u => string.Equals(u.Name, name, StringComparison.OrdinalIgnoreCase));
```

Comment:
All those `_users.Values` could use the nicer `GetUsers()`. But that's my 2 cents as a non C# dev